cobbler/remote.py

```diff
@@ -1354,6 +1443,22 @@ def remove_item(self, what: str, name: str, token: str, recursive: bool = True):
         :return: True if the action was successful.
         """
         self._log("remove_item (%s, recursive=%s)" % (what, recursive), name=name, token=token)
+        if token in self.transactions:
+            try:
+                obj_handle = self.get_item_handle(what, name, token)
+                obj = self.__get_object(obj_handle, token)
+            except ValueError:
+                return False
+            except CX:
+                return False
+            self.check_access(token, "remove_%s" % what, obj)
```

Comment:
So ideally, we'd not use lazy string formatting outside of logging, but this particular instance is IMHO quite confusing. WDYT about:

```suggestion
            self.check_access(token, f"remove_{what}", obj)
```